src/Base/Quantity.h

```diff
@@ -244,6 +244,8 @@ class BaseExport Quantity
     static Quantity KiloNewton;
     static Quantity MegaNewton;
 
+    static Quantity NewtonPerMeter;
+    
     static Quantity Pascal;
     static Quantity KiloPascal;
```

Comment:
What is the typical prefix? I mean if the typical use case is e.g. kN/m, then you should also define this here